docs/tutorials/build_a_node_in_rust.md

```diff
@@ -727,15 +761,15 @@ let persister = YourDataPersister { data_dir: ldk_data_dir.clone() };
 
 **References:** [`Persister` docs](https://docs.rs/lightning-background-processor/*/lightning_background_processor/trait.Persister.html)
 
-### 18. Start Background Processing
+### Step 19. Start Background Processing
 
 **What it's used for:** running tasks periodically in the background to keep LDK operational
 
 **Example:**
 ```rust
 let background_processor = BackgroundProcessor::start(
 	persister,
-	handle_event_callback,
+	&invoice_payer,
 	&chain_monitor,
 	&channel_manager,
 	&net_graph_msg_handler
```

Comment:
@TheBlueMatt Unrelated to this PR, but a bigger concern here is that I *think* these actually need to be `Arc`s rather than references, IIUC. Does that sound right?